tests/unit/parsers/test.manifestjson.js

```diff
@@ -2736,4 +2738,71 @@ describe('ManifestJSONParser', () => {
       return testHomepageUrl('http://test.org', true);
     });
   });
+
+  describe('@mdn/browser-compat-data', () => {
+    const usesArrayValueForFirefoxDesktopCompat = {
+      keys: ['browser_specific_settings'],
```

Comment:
I believe this is fine because we do not check the `browser_specific_settings` prop, we only check the props embedded in `browser_specific_settings` (e.g., `strict_min_version`, etc.)